.cursor/skills/waiting-for-build/scripts/fetch-build.py

```
@@ -0,0 +1,313 @@
+    sd = shutil.which("slackdump")
+    if not sd:
+        print("ERROR: slackdump not found. Install with: brew install slackdump", file=sys.stderr)
+        sys.exit(1)
+    return sd
+
+
+def _ensure_workspace(workspace: str, token: str, cookie: str) -> None:
+    """Authenticate slackdump workspace if not already set up."""
+    sd = _slackdump_bin()
+
+    result = subprocess.run([sd, "workspace", "list"], capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)